jx_sqlite/insert_table.py

```diff
@@ -335,3 +337,32 @@ def _insert(self, collection):
             )
 
             self.db.execute(prefix + records)
+        
+        commands = [cc for c in commands for cc in c]
+        for command in commands:
+            self.db.execute(command[0])
+            for d in command[1]:
+                del_column = []
+                for i, c in enumerate(self.sf.columns):
+                    if c.es_column==d.es_column and c.es_index==d.es_index:
+                        del_column.append(i)
+                        self.remove_col_from_db(d)
+                for i in del_column:
+                    del(self.sf.columns[i])
+                    
+                    
+    def remove_col_from_db(self, col):
```

Comment:
move this function to the Snowflake; keep all the database schema changes going through the Snowflake